kubernetes/websocket/wsclient.c

```diff
@@ -2,7 +2,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#ifndef _WIN32
 #include <unistd.h>
+#endif
```

Comment:
The code change of this file is OK to me.
But you need to rebase/merge the new code change from the upstream repo because I merged a PR today.